policy_packs/aws/backup/enforce_ebs_volume_backups/README.md

```diff
@@ -0,0 +1,183 @@
+      iam_role_arn = "arn:aws:iam::{{ $.account.id }}:role/turbot/core/guardrails_backup_service_role"
+      name         = "guardrails-ebs-resource-assignment"
+      plan_id      = aws_backup_plan.guardrails_ebs_backups.id
+      resources = ["arn:aws:ec2:*:*:volume/*"]
+    }
+```
+- Commit the updated policy setting via Terraform then let the `AWS > Backup > Stack` make the changes. 
+
+
+## Decommission the Backup Vault
+To completely remove a Backup Vault and all backups, execute the following steps:
+
+- Edit the Backup > Stack > Source policy setting in the policy pack.
+  - To flush the current backups, set the retention period to one day, as shown below. 
+  - Remove the Backup Selection from the `AWS > Backup > Stack > Source`.  This will prevent new backups from happening.
+
+When the changes are complete the `template` attribute for `Backup > Stack > Source` should look like this. 
+```hcl
+    |
+    resource "aws_backup_vault" "vault" {
+      name = "guardrails-backup-vault"
+      tags = {
+        turbot_version = "v5"
+      }
+    }
+    resource "aws_backup_plan" "guardrails_ebs_backups" {
+      name = "guardrails-backup-plan"
+      rule {
+        # The time allowed for the job to start, any longer and it will be cancelled.
+        start_window = 480
+        # The amount of time allowed for the backup to complete, before it is cancelled.
+        completion_window = 10080
+        #
+        rule_name         = "guardrails-ebs-backups-rule"
+        schedule          = "cron(0 5 ? * * *)"
+        target_vault_name = aws_backup_vault.vault.name
+        lifecycle {
+          delete_after = 1
+        }
+      }
+    }
+```
+
+- Wait a day or two for the backups to be flushed from the Guardrails Backup vault.  AWS will not allow a vault to be destroyed when backups are still present. 
+- This is most easily done by replacing `template` and `template_input` attributes of `Backup > Stack > Source` as shown below. The `[]` value instructs the `Backup > Stack` control to destroy all managed resources. Apply the changes to the Policy Pack.
+```hcl
+# AWS > Backup > Stack > Source
+resource "turbot_policy_setting" "aws_backup_stack_source" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-backup#/policy/types/backupStackSource"
+  value    = "[]"  
+}
+```
+- Verify that all the `Backup > Stack` controls have gone into an `ok` state.  If they go into `error`, the `Backup > Stack` control logs should give a good indication of what's wrong.
+- Set the `AWS > IAM > Stack > Source` policy to the below Terraform:  This will remove the Backup IAM role.
+```hcl
+# AWS > IAM > Stack > Source
+resource "turbot_policy_setting" "aws_iam_iam_stack_source" {
+  resource = turbot_policy_pack.main.id
+  type     = "tmod:@turbot/aws-iam#/policy/types/iamStackSource"
+  value    = "[]"
+}
+```
+- Verify that all the `IAM > Stack` controls are in `ok`.
+- Remove the policy pack from your Guardrails workspace. 
```

Comment:
Should users run `terraform destroy` at this point? Do they need to do anything in Guardrails first, like detach? Or can they just delete the policy pack?